src/shared/nt-results/ResultsByLoadContainer.react.js

```diff
@@ -84,37 +84,41 @@ class ResultsByLoadContainer extends React.Component {
       .on('drag', this.dragged)
     )
 
-    this.setState({ x: this.container.clientWidth / 2 })
+    if (this.container) {
+      this.setState({ x: this.container.clientWidth / 2 })
+    }
   }
 
   dragged = () => {
-    this.setState({ x: d3.event.x < 0 ? 0 : d3.event.x })
+    const x = d3.event.x
+    this.setState({
+      x: x < 0 ? 0 : x > this.container.clientWidth
+        ? this.container.clientWidth : x
```

Comment:
I don't think so, but I didn't mean what I write is good. Gonna look for better way to write this.